ironic/drivers/modules/drac/raid.py

```diff
@@ -727,6 +765,45 @@ def _filter_logical_disks(logical_disks, include_root_volume,
     return filtered_disks
 
 
+def _change_physical_disk_conversion(
+        node, mode=None, controllers_to_physical_disk_ids=None):
```

Comment:
Can you remove the controllers_to_physical_disk_ids parameter in the line above to simplify things?

Reply:
And remove it from the comment below.

Reply:
Done.